gpu/gpudraw/drawer.go

```diff
@@ -0,0 +1,147 @@
+// Copyright 2024 Cogent Core. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package gpudraw
+
+//go:generate core generate
+
+import (
+	"image"
+	"image/draw"
+	"sync"
+
+	"cogentcore.org/core/gpu"
+)
+
+// These draw.Op constants are provided so that users of this package don't
+// have to explicitly import "image/draw".  We also add the fill operation.
+const (
+	Over = draw.Over
+	Src  = draw.Src
+	Fill = draw.Src + 1
+)
+
+// AllocChunk is number of images / matrix elements to allocate
+// at a time, to reduce number of reallocations.
+// Should be set to the rough scale of number of items expected.
+var AllocChunk = 16
```

Comment:
Is `Chunk` better than something like `Batch`?

Reply:
no.